subprojects/core/src/main/java/org/gradle/internal/service/scopes/CrossBuildSessionScopeServices.java

```diff
@@ -91,13 +92,18 @@ BuildOperationListenerManager createBuildOperationListenerManager() {
     }
 
     BuildOperationExecutor createBuildOperationExecutor() {
+        // Wrap to prevent exposing Stoppable, as we don't want to stop at this scope
```

Comment:
Why is this service declaration here at all instead of it coming from the parent?